tools/eeprom-tool/cli_helper.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * BSD 3-Clause License
+ *
+ * Copyright (c) 2020, Analog Devices, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from
+ *    this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+#ifndef CLI_HELPER_H
+#define CLI_HELPER_H
+
+#include <getopt.h>
+#include <iostream>
+#include <stdio.h>
+#include <string>
+
+#include <aditof/connections.h>
+#include <aditof/device_interface.h>
+#include <aditof/status_definitions.h>
+
+using namespace aditof;
+using namespace std;
+
+enum ActionType { WRITE, READ, LIST_EEPROMS, UNKNOWN };
+
+typedef struct {
+    string path;
+    string ip = "0.0.0.0";
+    string eepromName = "";
+    ConnectionType connectionType = ConnectionType::LOCAL;
+    ActionType actionType = UNKNOWN;
+    bool isConnectionSpecifed = false;
+} CLIArguments;
+
+void printHelpMessage();
+Status parseArguments(int argc, char *argv[], CLIArguments &cliArguments);
+
+#endif // CLI_HELPER_H
```

Comment:
This is still missing a new line. I've seen other files as well.

Reply:
fixed